features/step_definitions/jitsi_steps.rb

```diff
@@ -0,0 +1,5 @@
+Then(/^I should see a start jitsi meet button$/) do
+  expect(page).to have_css '#start-jitsi'
+  # src = page.find(:css, '.btn-jitsi a')['src']
```

Comment:
Ah! Got it! It should be "Repeat Scrum" not "Scrum" because "Scrum" will run only for once.